charts/argo-workflows/Chart.yaml

```diff
@@ -16,5 +16,5 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
-    - kind: added
-      description: Add parameters for tuning revisionHistoryLimit and emptyDir volumes.
+    - kind: fixed
+      description: Aligne version label
```

Comment:
nit: it's "align"

Reply:
let me resign an engineer. 👴

Reply:
You can resign after there are no more PRs needed.

Reply:
I fixed in https://github.com/argoproj/argo-helm/pull/2342/commits/d84f766889a39c9b08c82031271fdf4cf39e4205 🙋

Reply:
lgtm!